test/e2e/export/simple_test.go

```diff
@@ -0,0 +1,60 @@
+package e2e_export_test
+
+import (
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/werf/werf/v2/test/pkg/werf"
+)
+
+var _ = Describe("Simple export", Label("e2e", "export", "simple"), func() {
+	It("should succeed and export images",
+		func() {
+			SuiteData.WerfRepo = strings.Join([]string{SuiteData.RegistryLocalAddress, SuiteData.ProjectName}, "/")
+			By("state0 running")
+			{
+				repoDirname := "repo0"
+				fixtureRelPath := "simple/state0"
+
+				By("state0: preparing test repo")
+				SuiteData.InitTestRepo(repoDirname, fixtureRelPath)
+
+				By("state0: running export")
+				werfProject := werf.NewProject(SuiteData.WerfBinPath, SuiteData.GetTestRepoPath(repoDirname))
+				exportOut := werfProject.Export(&werf.ExportOptions{
+					CommonOptions: werf.CommonOptions{
+						ExtraArgs: []string{
+							"--dev",
+							"--tag",
+							SuiteData.WerfRepo,
+						},
+					},
+				})
+				Expect(exportOut).To(ContainSubstring("Exporting image..."))
+			}
+			By("state1 running")
+			{
+				repoDirname := "repo0"
+				fixtureRelPath := "simple/state1"
+
+				By("state1: preparing test repo")
+				SuiteData.InitTestRepo(repoDirname, fixtureRelPath)
+
+				By("state2: running export multiplatform image")
+				werfProject := werf.NewProject(SuiteData.WerfBinPath, SuiteData.GetTestRepoPath(repoDirname))
+				SuiteData.Stubs.SetEnv("WERF_REPO", SuiteData.WerfRepo)
+				exportOut := werfProject.Export(&werf.ExportOptions{
+					CommonOptions: werf.CommonOptions{
+						ExtraArgs: []string{
+							"--dev",
+							"--tag",
+							SuiteData.WerfRepo,
+						},
+					},
+				})
+				Expect(exportOut).To(ContainSubstring("Exporting image..."))
```

Comment:
Let's check that the final image is built for specified platforms by checking [index manifest](https://docs.docker.com/build/building/multi-platform/):

Example:
```
$ crane manifest localhost:5001/test/test | jq
{
  "schemaVersion": 2,
  "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
  "manifests": [
    {
      "mediaType": "application/vnd.oci.image.manifest.v1+json",
      "size": 412,
      "digest": "sha256:601cd229e281c1637f64902ffec9f7ed3bd1a42514a4dab85e0850731655de8d",
      "platform": {
        "architecture": "arm64",
        "os": "linux"
      }
    },
    {
      "mediaType": "application/vnd.oci.image.manifest.v1+json",
      "size": 412,
      "digest": "sha256:b2bd66ea29c45f1789d46a9a89f9452f15c5415235c689fb2f5cc47dcea8d82b",
      "platform": {
        "architecture": "arm64",
        "os": "linux"
      }
    }
  ]
}
```

You can see [here](https://github.com/werf/werf/blob/49a1540031ed3733b809a348722ccef631553c98/pkg/docker_registry/api.go#L147-L179) how to work with index manifest.

Reply:
+ check absence of werf labels